config/nrfconnect/chip-module/CMakeLists.txt

```diff
@@ -170,10 +176,22 @@ if (CONFIG_CHIP_OPENTHREAD_CONFIG)
     zephyr_set_openthread_config(${CHIP_OPENTHREAD_CONFIG})
 endif()
 
+# Find required programs
+
+find_program(GN_EXECUTABLE gn)
+if (${GN_EXECUTABLE} STREQUAL GN_EXECUTABLE-NOTFOUND)
+    message(FATAL_ERROR "The 'gn' command was not found. Make sure you have GN installed.")
+    return()
+endif()
+
 # ==============================================================================
 # Generate configuration for CHIP GN build system
 # ==============================================================================
 
+if (CONFIG_CHIP_PW_RPC)
+    chip_gn_arg_import("${GN_ROOT_TARGET}/lib/pw_rpc/pw_rpc.gni")
```

Comment:
fiy: This should be removed anyway once pigweed-app is refactored to import pw_rpc in CMake directly.